src/zope/testing/stringliterals.py

```diff
@@ -0,0 +1,50 @@
+import doctest
+import re
+
+ALLOW_UNICODE = doctest.register_optionflag('ALLOW_UNICODE')
+ALLOW_BYTES = doctest.register_optionflag('ALLOW_BYTES')
+
+
+class LiteralsOutputChecker(doctest.OutputChecker):
+    """A checker with support for u/b literals.
+
+    Use the option flags from this module to get the desired effects:
+        ALLOW_UNICODE removes u/U prefixes
+        ALLOW_BYTES   removes b/B prefixes
+
+    This version was copied from pytest
+        https://github.com/pytest-dev/pytest/blob/master/_pytest/doctest.py
+    which commented:
+
+    Copied from doctest_nose_plugin.py from the nltk project:
+        https://github.com/nltk/nltk
+    Further extended to also support byte literals.
```

Comment:
This raises licensing questions that make me want to stay far away from this pull request.

Reply:
It's not the spirit, but the wording of the ZF Committer Agreement that makes me pause (ZPL only, unless ZF declares in writing another acceptable license; Committer represents and warrants that the Committer has legal authority over Contributed Code, whatever that means).  I'm allergic to legalese.